src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -46,7 +46,10 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
 
         commandResult = command.execute(model);
 
-        if (!(command instanceof UndoCommand || command instanceof RedoCommand)) {
+        boolean canBeUndone = !(command instanceof UndoCommand || command instanceof RedoCommand
+                || command instanceof HelpCommand );
```

Comment:
Great job factoring this long condition out

Minor issue but HelpCommand and the closing bracket no space btw